cfme/tests/infrastructure/test_iso_provisioning.py

```diff
@@ -52,9 +52,10 @@ def pytest_generate_tests(metafunc):
 
 
 @pytest.fixture
-def datastore_init(iso_cust_template, iso_datastore, provisioning):
+def datastore_init(appliance, iso_cust_template, iso_datastore, provisioning):
     if not iso_datastore.exists():
-        iso_datastore.create()
+        collection = appliance.collections.iso_datastores
+        collection.create(iso_datastore)
```

Comment:
@psav : This is the PR I was talking about, I am not able to understand where iso_datastore value is coming from? and where I have to make changes to make it work.